english/music-festival/music-festival.md

```diff
@@ -0,0 +1,7 @@
+Welcome to the heart of your most fondly remembered music festival.
+You don't need a ticket.  You've been here before.
+
+Where are you?
+
+[Woodstock 1994](woodstock-1994.md)
+[Lollapalooza 1997](lollapalooza-1997.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.